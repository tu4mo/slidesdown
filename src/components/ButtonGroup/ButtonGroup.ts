import styled from 'styled-components'

const ButtonGroup = styled.div`
  display: flex;

  & > *:not(:last-child) {
    margin-right: 1rem;
  }
`

export default ButtonGroup
