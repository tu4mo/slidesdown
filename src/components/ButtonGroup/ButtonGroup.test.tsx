import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import ButtonGroup from './ButtonGroup'

it('renders correctly', () => {
  const tree = renderer.create(<ButtonGroup>Test</ButtonGroup>).toJSON()

  expect(tree).toMatchSnapshot()
})
