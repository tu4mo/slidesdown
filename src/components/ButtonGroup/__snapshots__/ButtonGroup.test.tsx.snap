// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 > *:not(:last-child) {
  margin-right: 1rem;
}

<div
  className="c0"
>
  Test
</div>
`;
